src/main/scala/org/allenai/scholar/Author.scala

```diff
@@ -0,0 +1,43 @@
+package org.allenai.scholar
+
+import StringUtils._
+import spray.json.DefaultJsonProtocol._
+
+case class Author(firstName: String, middleNames: Seq[String], lastName: String) {
```

Comment:
This is good for now but tricky to handle correctly in the long run. E.g. John Smith Jr. (Jr. is not last name), Nando de Freitas (de is not middle name). Factorie has a collection of name wrangling utilities we may consider using.